From b5c2e417b2c4e91327c3112a781c8e4a0526abfc Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sat, 18 Apr 2020 21:22:26 +0300
Subject: Revert "Add suspend_resume trace events to the atrace
 'freq' category."

This reverts commit 581c22f979af05e48ad4843cdfa9605186d286da.

  disable this changes, because on some phones not worked deep sleep
  based on the patch by Hamido
---
 cmds/atrace/atrace.cpp | 1 -
 cmds/atrace/atrace.rc  | 2 --
 2 files changed, 3 deletions(-)

diff --git a/cmds/atrace/atrace.cpp b/cmds/atrace/atrace.cpp
index 1429bc8b0..a4b00f8e0 100644
--- a/cmds/atrace/atrace.cpp
+++ b/cmds/atrace/atrace.cpp
@@ -171,7 +171,6 @@ static const TracingCategory k_categories[] = {
         { OPT,      "events/clk/clk_disable/enable" },
         { OPT,      "events/clk/clk_enable/enable" },
         { OPT,      "events/power/cpu_frequency_limits/enable" },
-        { OPT,      "events/power/suspend_resume/enable" },
     } },
     { "membus",     "Memory Bus Utilization", 0, {
         { REQ,      "events/memory_bus/enable" },
diff --git a/cmds/atrace/atrace.rc b/cmds/atrace/atrace.rc
index 6e460a0d2..f1426b6bd 100644
--- a/cmds/atrace/atrace.rc
+++ b/cmds/atrace/atrace.rc
@@ -49,8 +49,6 @@ on late-init
     chmod 0666 /sys/kernel/tracing/events/power/cpu_frequency_limits/enable
     chmod 0666 /sys/kernel/debug/tracing/events/power/gpu_frequency/enable
     chmod 0666 /sys/kernel/tracing/events/power/gpu_frequency/enable
-    chmod 0666 /sys/kernel/debug/tracing/events/power/suspend_resume/enable
-    chmod 0666 /sys/kernel/tracing/events/power/suspend_resume/enable
     chmod 0666 /sys/kernel/debug/tracing/events/cpufreq_interactive/enable
     chmod 0666 /sys/kernel/tracing/events/cpufreq_interactive/enable
     chmod 0666 /sys/kernel/debug/tracing/events/vmscan/mm_vmscan_direct_reclaim_begin/enable
-- 
2.17.1

